### PR TITLE
#73-challenge-is-even

### PR DESCRIPTION
**Pull request by Portaltocoding**
### Debugging in different cases 
<p>Implementing not just the filter method of a RegEX but also a  switch statement </p>

### CODE REVIEW

File: staff/carlos-portal/challenge/73-is-even.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+let result5 = isEven('adios')
+console.assert(result5 === false, {result: result5, message: 'Test 5 no pasado'})
+
+let result6 = isEven(Number.MAX_SAFE_INTEGER)
+console.assert(result6 === false, {result: result6, message: 'Test 6 no pasado'})
+
+let result7 = isEven(Number.MAX_SAFE_INTEGER + 1)
+console.assert(result7 === true, {result: result7, message: 'Test 7 no pasado'})
+
+}
```
Comment (CarlosLemonCode) [inline staff/carlos-portal/challenge/73-is-even.js]: The last curly bracket is giving me an error.

**Comment by CarlosLemonCode** [comment]
I can't see the link to the _issue_! :(

**Review by Carlos00004 — CHANGES_REQUESTED** [review]
Removing the last curly bracket and adding the link to the issue would be enough. Good job! 👍

File: staff/carlos-portal/challenge/73-is-even.js (hunk 1 above)
Comment (Rafa0297) [inline staff/carlos-portal/challenge/73-is-even.js]: Delete curly brace from line 39

**Review by Rafa0297 — CHANGES_REQUESTED** [review]

**Review by itsmePo — APPROVED** [review]
After deleting the cbrace, the code works perfectly fine! :D

**Comment by itsmePo** [comment]
Everything works! :D
Alt Text

**Comment by Portaltocoding** [comment]
https://github.com/b00tc4mp/isdi-parttime-202406/issues/73#issue-2394117756 

link a la issue 🍰

**Review by venturars — APPROVED** [review]

**Pull request merged**